packages/jest-environment-node/src/index.js

```diff
@@ -37,12 +37,17 @@ class NodeEnvironment {
     this.fakeTimers = new FakeTimers(global, this.moduleMocker, config);
   }
 
-  dispose() {
```

Comment:
It's a breaking change to not run them if specified in custom environments, isn't it?

Reply:
Yep, you're right. I'm fine with it though, as long as it lands in the changelog.